release.md

```diff
@@ -55,13 +55,16 @@
 
 1. Press `Release this version` in App Store Connect
 1. Tag the release commit and create a GitHub release. Attach the binary as an artefact to the GitHub release (if you're using the automated release command, you can find the binary in the Artifacts top section in the CI build).
-1. Release new version for the Babylon SDK:
-	1. Ask SDK team (#sdk_squad) about the SDK version number
-	1. Trigger a hockey build from that branch using its command (eg. `/distribute_sdk version:0.1.11 pre_release:false branch:release/3.7.0`) in `#ios-build`.
 1. Merge the changes back to develop.
 
+### 3. SDK Release
 
-### 3. Release calendar
+1. Ask SDK team (#sdk_squad) about the SDK version number.
+2. Follow the [Internal SDK Release Process](https://engineering.ops.babylontech.co.uk/docs/cicd-deployments/#mobile-sdk-releases-ios-android) to open a CR ticket on the CRP board.
+3. Trigger a hockey build from that branch using its command (eg. `/distribute_sdk version:0.1.11 pre_release:false branch:release/3.7.0`) in `#ios-build`.
```

Comment:
`pre_release` argument is no longer needed. It's determined now based on `version`, e.g. `0.1.11` is release and `0.1.11-beta` is prerelease.

Reply:
I will update the command as soon as we remove the legacy. I think you have created a story for that

Reply:
Any update on this?

Reply:
The command and process has been udpated